api/v1beta1/aerospikecluster_types.go

```diff
@@ -62,6 +62,8 @@ type AerospikeClusterSpec struct {
 	PodSpec AerospikePodSpec `json:"podSpec,omitempty"`
 	// SeedsFinderServices describes services which are used for seeding Aerospike nodes.
 	SeedsFinderServices SeedsFinderServices `json:"seedsFinderServices,omitempty"`
+	// SecurityContext that will be added to aerospike-server container created by operator.
+	AerospikeSecurityContext *corev1.SecurityContext `json:"aerospikeSecurityContext,omitempty"`
```

Comment:
This can be moved to  PodSpec -> AerospikeContainerSpec -> SecurityContext
---
struct PodSpec {
  .
  AerospikeContainerSpec AerospikeContainerSpec `json:"aerospike,omitempty"`
  .
}


struct AerospikeContainerSpec {
   SecurityContext *coverv1.SecurityContext
}
---

This struct can then be used if in the future we want to allow more container-specific properties to be configured.

Reply:
Updated